src/climate_indices/indices.py

```diff
@@ -315,6 +316,35 @@ def spei(
                 skews,
             )
 
+    elif distribution is Distribution.fisk:
+
+        # get (optional) fitting parameters if provided
+        if fitting_params is not None:
+            probabilities_of_zero = fitting_params["probabilities_of_zero"]
+            locs = fitting_params["locs"]
+            scales = fitting_params["scales"]
+            c = fitting_params["c"]
```

Comment:
This is going to conflict a bit with #453.

Reply:
Yes, we should/will normalize.